dojo/group/urls.py

```diff
@@ -3,11 +3,11 @@
 from dojo.group import views
 
 urlpatterns = [
-    re_path(r'^group$', views.group, name='groups'),
-    re_path(r'^group/(?P<gid>\d+)$', views.view_group, name='view_group'),
-    re_path(r'^group/(?P<gid>\d+)/edit$', views.edit_group, name='edit_group'),
-    re_path(r'^group/(?P<gid>\d+)/delete', views.delete_group, name='delete_group'),
-    re_path(r'^group/add$', views.add_group, name='add_group'),
+    re_path(r'^group$', views.ListGroups.as_view(), name='groups'),
+    re_path(r'^group/add$', views.AddGroup.as_view(), name='add_group'),
+    re_path(r'^group/(?P<group_id>\d+)$', views.ViewGroup.as_view(), name='view_group'),
+    re_path(r'^group/(?P<group_id>\d+)/edit$', views.EditGroup.as_view(), name='edit_group'),
+    re_path(r'^group/(?P<group_id>\d+)/delete', views.DeleteGroup.as_view(), name='delete_group'),
```

Comment:
I'm pretty indifferent on the `$` termination on these patterns, but we should probably be consistent across these?

Reply:
Consistency would be good. The inconsistency was there from before (the diff is a liar 🤔) but I can update the URLs to all match